scripts/publish_vessel_info

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+source pipe-tools-utils
+
+THIS_SCRIPT_DIR="$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )"
+ASSETS=${THIS_SCRIPT_DIR}/../assets
+ARGS=( \
+  SOURCE \
+  TEMP_BUCKET \
+  ELASTIC_SEARCH_SERVER_URL \
+  ELASTIC_SEARCH_SERVER_AUTH \
+  ELASTIC_SEARCH_INDEX_ALIAS \
+)
+
+################################################################################
+# Validate and extract arguments
+################################################################################
+display_usage() {
+  echo -e "\nUsage:\n$0 SOURCE TEMP_BUCKET ELASTIC_SEARCH_SERVER_URL ELASTIC_SEARCH_SERVER_AUTH ELASTIC_SEARCH_INDEX_ALIAS\n"
+}
+
+if [[ $# -ne ${#ARGS[@]} ]]
+then
+    display_usage
+    exit 1
+fi
+
+ARG_VALUES=("$@")
+PARAMS=()
+for index in ${!ARGS[*]}; do
+  echo "${ARGS[$index]}=${ARG_VALUES[$index]}"
+  declare "${ARGS[$index]}"="${ARG_VALUES[$index]}"
+done
+
+
+TEMP_PATH=gs://${TEMP_BUCKET}/pipe-vessels/$( date -u "+%FT%T.%N" )
+
+echo "Running $0"
+echo "  SOURCE: $SOURCE"
+echo "  TEMP_BUCKET: $TEMP_BUCKET"
+echo "  ELASTIC_SEARCH_SERVER_URL: $ELASTIC_SEARCH_SERVER_URL"
+echo "  ELASTIC_SEARCH_SERVER_AUTH: [HIDDEN]"
+echo "  ELASTIC_SEARCH_INDEX_ALIAS: $ELASTIC_SEARCH_INDEX_ALIAS"
+################################################################################
+# Export records to json files
+################################################################################
+echo "Exporting records from $SOURCE"
+EXTRACT_PATH=$TEMP_PATH/bq/*.json
+bq extract \
+  --destination_format=NEWLINE_DELIMITED_JSON \
+  $SOURCE \
+  $EXTRACT_PATH
+if [ "$?" -ne 0 ]; then
+  echo "  Unable to extract ${SOURCE} to ${EXTRACT_PATH}"
+  exit 1
+fi
+echo "  Exported records from ${SOURCE} to ${EXTRACT_PATH}"
+
+################################################################################
+# Download files locally
+################################################################################
+echo "Downloading records from ${SOURCE} to local disk"
+LOCAL_JSON_PATH=./data/json/
+mkdir -p ${LOCAL_JSON_PATH}
+gsutil -m cp ${EXTRACT_PATH} ${LOCAL_JSON_PATH}
+if [ "$?" -ne 0 ]; then
+  echo "  Unable to download records data locally from ${TEMP_BUCKET}"
+  exit 1
+fi
+echo "  Downloaded records from ${TEMP_BUCKET}"
+
+################################################################################
+# Load data into Elastic Search
+################################################################################
+echo "Loading data into Elastic Search"
+INDEX_SCHEMA=${ASSETS}/elasticsearch/vessels.json
+cat ${LOCAL_JSON_PATH}/*.json | python -m pipe_vessels.elasticsearch.importer $ELASTIC_SEARCH_SERVER_URL $ELASTIC_SEARCH_SERVER_AUTH $ELASTIC_SEARCH_INDEX_ALIAS $INDEX_SCHEMA
```

Comment:
@andres-arana I recall having some issues with logging when running python. I fixed this using `-u     : unbuffered binary stdout and stderr;` argument.